Sources/Extensions/SwiftStdlib/ComparableExtensions.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ComparableExtensions.swift
+//  SwifterSwift
+//
+//  Created by Shai Mishali on 5/4/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+// MARK: - Methods
+public extension Comparable {
+    /// SwifterSwift: Returns true if value is between provided `min`
+    /// and `max`.
+    ///
+    ///    1.isBetween(min: 5, max: 7) // false
+    ///    7.isBetween(min: 6, max: 12) // true
+    ///    date.isBetween(min: date1, max: date2)
+    ///    "c".isBetween(min: "a", max: "d") // true
+    ///    0.32.isBetween(min: 0.31, max: 0.33) // true
+    ///
+    /// - parameter min: Minimum comparable value.
+    /// - parameter max: Maximum comparable value.
+    ///
+    /// - returns: `true` if value is between `min` and `max`, `false` otherwise.
+    func isBetween(min: Self, max: Self) -> Bool {
```

Comment:
@freak4pc Would you mind explicitly marking these two extensions as `public`? I know they're already public due to the scope of the extension, but for the sake of consistency in our codebase.

Reply:
Done & rebased